### ISSUE TITLE
🐛 BUG: Empty <pre> in Markdown throws

### ISSUE THREAD

**Issue by Merri** [snowpackjs/astro#1280]
### What package manager are you using?

pnpm

### What operating system are you using?

Windows

### Describe the Bug

Markdown page generation fails with `TypeError: Cannot read property 'type' of undefined` when a page contains empty `<pre>` element.

The page works fine as long as the element contains more than a single line change.

Expected behavior is to just work even with no content in the `pre` element.

### Steps to Reproduce

1. Create a Markdown page such as `index.md`
2. Add in `<pre></pre>` (can also be separated by a single line change)

**Comment by jasikpark** [comment]
I got this error while reproducing https://codesandbox.io/s/astro-1280-reproduction-x5d0s: 

I set `yarn dev` to run with `astro dev --verbose` to get the output!

```sh
[executing astro] TypeError: Cannot read property 'type' of undefined
    at file:///sandbox/node_modules/@astrojs/markdown-support/dist/codeblock.js:52:16
    at overload (file:///sandbox/node_modules/unist-util-visit/index.js:68:16)
    at node (element<pre>) (file:///sandbox/node_modules/unist-util-visit/node_modules/unist-util-visit-parents/index.js:124:31)
    at node (root) (file:///sandbox/node_modules/unist-util-visit/node_modules/unist-util-visit-parents/index.js:141:79)
    at visitParents (file:///sandbox/node_modules/unist-util-visit/node_modules/unist-util-visit-parents/index.js:81:30)
    at visit (file:///sandbox/node_modules/unist-util-visit/index.js:60:7)
    at file:///sandbox/node_modules/@astrojs/markdown-support/dist/codeblock.js:44:5
    at wrapped (file:///sandbox/node_modules/trough/index.js:111:16)
    at next (file:///sandbox/node_modules/trough/index.js:62:23)
    at done (file:///sandbox/node_modules/trough/index.js:145:7)
    at then (file:///sandbox/node_modules/trough/index.js:155:5)
    at wrapped (file:///sandbox/node_modules/trough/index.js:133:9)
    at next (file:///sandbox/node_modules/trough/index.js:62:23)
    at done (file:///sandbox/node_modules/trough/index.js:145:7)
    at then (file:///sandbox/node_modules/trough/index.js:155:5)
    at wrapped (file:///sandbox/node_modules/trough/index.js:133:9) {
  __snowpackBuildDetails: { name: 'snowpack-astro', step: 'load' }
}
```

![image](https://user-images.githubusercontent.com/10626596/132388411-cfc4846e-f6d9-488a-955d-4a596e8b374c.png)

**Comment by jasikpark** [comment]
Created a PR to fix: #1332

**Comment by jasikpark** [comment]
Fixed in #1332
